packages/moonstone/ProgressBar/ProgressBarTooltip.js

```diff
@@ -135,39 +258,54 @@ const ProgressBarTooltipBase = kind({
 
 			return children;
 		},
-		className: ({orientation, proportion, side, styler}) => {
-			side = getSide(orientation, side);
+		className: ({orientation, position, proportion, side, styler}) => {
+			const [h, v] = getSide(orientation, side, position);
 
 			return styler.append(
 				orientation,
 				{
-					afterMidpoint: proportion > 0.5,
-					ignoreLocale: side === 'left' || side === 'right'
-				},
-				(side === 'before' || side === 'left') ? 'before' : 'after'
+					above: (v === 'above'),
+					below: (v === 'below'),
+					before: (h === 'before'),
+					after: (h === 'after'),
+					left: (h === 'left' || (h === 'auto' && proportion <= 0.5)),
+					right: (h === 'right' || (h === 'auto' && proportion > 0.5))
+				}
 			);
 		},
-		arrowAnchor: ({proportion, orientation}) => {
+		arrowAnchor: ({orientation, position, proportion, rtl}) => {
 			if (orientation === 'vertical') return 'middle';
-			return proportion > 0.5 ? 'left' : 'right';
+
+			const [h] = getSide(orientation, false, position);
+			switch (h) {
+				case 'auto':
+					return proportion > 0.5 ? 'left' : 'right';
+				case 'before':
+					return rtl ? 'right' : 'left';
```

Comment:
We'll need to communicate this new RTL ability in the release notes.  It should probably also be called out above in the description (at least, when we suppress the default behavior).

Reply:
It is vaguely noted but agree it should be expanded.